ios/RNFetchBlobNetwork.m

```diff
@@ -169,7 +170,8 @@ - (void) sendRequest:(__weak NSDictionary  * _Nullable )options
     self.expectedBytes = 0;
     self.receivedBytes = 0;
     self.options = options;
-    
+
```

Comment:
I'm happy for you to use your discretion here, but my personal preference would be to avoid this kind of formatting change in the PR back upstream. (Not that upstream actually has consistent whitespace conventions; more that in general small, focused patches are more resilient to conflicts.)